lib/email/receiver.rb

```diff
@@ -107,7 +107,7 @@ def process!
 
           Email::Validator.ensure_valid!(@mail)
 
-          @from_email, @from_display_name = parse_from_field
+          @from_email, @from_display_name = parse_from_field(@mail)
```

Comment:
Made this argument explicit since there was no need for it to be ~~magic~~ implicit